### DISCUSSION TITLE
Global configuration file for supported chains

### DISCUSSION THREAD

**Discussion by schmidsi** [graphprotocol/graph-tooling#1061] [Ideas]
Maybe it would make sense to consolidate the chain configs across different codes in one global file that looks something like this:
```
const chainList = {
  mainnet: {
    name: 'Ethereum',
    chainId: 1,
    shortName: 'eth',
    blockExplorerType: 'etherscan',
    blockExplorerAPIUrl: 'https://etherscan.io',
    jsonRpcUrl: 'https://rpc.ankr.com/eth',
  }
}
```

We might want to reuse this here: https://github.com/DefiLlama/chainlist/blob/main/constants/extraRpcs.js. Thoughts? CC @saihaj

**Comment by saihaj** [comment]
I agree we should refactor the `chainNames`, and different configs as constants and simplify the code

**Reply by PaulRBerg** [reply]
Any updates on this, @saihaj?

Potentially relevant:

https://github.com/sablier-labs/deployments/blob/211bdadfcea5d7ec18f02c06096774220fbd33e9/src/chains/mainnets.ts
